packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx

```diff
@@ -104,6 +114,8 @@ export const ChatMessageComponentAsMessageBubble = (props: ChatMessageComponentA
           onEditClick={onEditClick}
           onRemoveClick={onRemoveClick}
           strings={strings}
+          messageReadByCount={message.readNumber ? message.readNumber : 0}
```

Comment:
nit, can use nullish coalescing operator here:
```suggestion
          messageReadByCount={message.readNumber ?? 0}
```

Reply:
Updated !